### PR TITLE
Fix TypeError in FIM Inventory using new error-handler

### LINKED ISSUE #5210: Uncaught (in promise) TypeError in Integrity monitoring inventory
| Wazuh | Elastic | Rev  | Security |
| ----- | ------- | ---- | -------- |
| 4.4.0   | 7.x     | 03| Basic, ODFE, Xpack |

| Browser |
| ------- |
| Firefox|

**Description**
When we access `Recent events` from an Integrity monitoring inventory file, we get:

```
Uncaught (in promise) TypeError: scope.updateQueryAndFetch is not a function
    _callee$ https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/wazuh/wazuh.chunk.10.js:5
    tryCatch https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/customImportMapDashboards/customImportMapDashboards.plugin.js:6
    invoke https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/customImportMapDashboards/customImportMapDashboards.plugin.js:6
    defineIteratorMethods https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/customImportMapDashboards/customImportMapDashboards.plugin.js:6
    fileDetail_asyncGeneratorStep https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/wazuh/wazuh.chunk.10.js:5
    _next https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/wazuh/wazuh.chunk.10.js:5
    promise callback*fileDetail_asyncGeneratorStep https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/wazuh/wazuh.chunk.10.js:5
    _next https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/wazuh/wazuh.chunk.10.js:5
    fileDetail_asyncToGenerator https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/wazuh/wazuh.chunk.10.js:5
    fileDetail_asyncToGenerator https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/wazuh/wazuh.chunk.10.js:5
    checkFilterManager https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/wazuh/wazuh.chunk.10.js:5
    openEventCurrentWindow https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/wazuh/wazuh.chunk.10.js:5
    FileDetails https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/wazuh/wazuh.chunk.10.js:5
    navigateToModule https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/wazuh/wazuh.chunk.10.js:5
    FileDetails https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/wazuh/wazuh.chunk.10.js:5
    onMouseDown https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/plugin/wazuh/wazuh.chunk.10.js:5
    s https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    f https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    m https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    m https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    at https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    ot https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    st https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    ft https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    D https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    F https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    Jt https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    Zt https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    unstable_runWithPriority https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:451
    $o https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    W https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    Gt https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    Vt https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    Yt https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    ht https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    cn https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    Ka https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    lc https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    uc https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    sc https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    Qs https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    Yo https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    unstable_runWithPriority https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:451
    $o https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    Yo https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    B https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:451
    onmessage https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:451
    __osdSharedDeps__ https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:451
    a https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:1
    __osdSharedDeps__ https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    a https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:1
    __osdSharedDeps__ https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:443
    a https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:1
    __osdSharedDeps__ https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:359
    a https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:1
    __osdSharedDeps__ https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:389
    a https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:1
    n https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:1
    __osdSharedDeps__ https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:1
    <anonymous> https://demo-440-rc1-wazuh-45bf2715ce73498d.elb.us-west-1.amazonaws.com/44003/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:1
```

**Preconditions**
1. Demo environment

**Steps to reproduce**
1. Navigate to `Integrity monitoring>Inventory`
2. Click on a file
3. Click on `Recent events`

**Expected Result**
1. No errors

**Actual Result**
1. `Uncaught (in promise) TypeError: scope.updateQueryAndFetch is not a function`

**Screenshots**

https://user-images.githubusercontent.com/80041853/218728757-1411d9e8-d49b-4598-a77f-e0bfd14ecd41.mp4

### PR DESCRIPTION
**Pull request by Machi3mfl**
### Description
This PR, solves and unhandled TypeError adding the new error management solution.

Closes #5210 

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: public/components/agents/fim/inventory/fileDetail.tsx [hunk 1]
```diff
@@ -228,22 +229,26 @@ export class FileDetails extends Component {
...
+        filterManager.addFilters([filters]);
+        const scope = await ModulesHelper.getDiscoverScope();
+        scope.updateQueryAndFetch && scope.updateQueryAndFetch({ query: null });
+      } else {
+        setTimeout(() => {
+          this.checkFilterManager(filters);
+        }, 200);
+      }
+    }catch(error){
+      ErrorHandler.handleError(error as Error);
```
[the remark this answers is not in the document]
Comment (Machi3mfl) [reply public/components/agents/fim/inventory/fileDetail.tsx:251]: If the error is a javascript native error class. The error will be shown in the browser console like a console error. 
Instead (https errors) the error details will be shown in a toast. 
Those are the forms of management defined up to now, we can add more or changes them

**Comment by Desvelao** [comment]
**question**: I am not sure why this pull request has so much commits. Did you use another base branch instead the target branch of the pull request to the fix/feature?

File: CHANGELOG.md [hunk 2]
```diff
@@ -65,6 +65,37 @@ All notable changes to the Wazuh app project will be documented in this file.
...
+- Fixed greyed and disabled menu section names [#5101](https://github.com/wazuh/wazuh-kibana-app/pull/5101)
+- Fixed mispelling in the NIST module [#5107](https://github.com/wazuh/wazuh-kibana-app/pull/5107)
+- Fixed Statistic cronjob bulk document insert [#5150](https://github.com/wazuh/wazuh-kibana-app/pull/5150)
+- Fixed the style of the buttons showing more event information in the event view table. [#5137](https://github.com/wazuh/wazuh-kibana-app/pull/5137)
+- Fixed Inventory module for solaris agents [#5144](https://github.com/wazuh/wazuh-kibana-app/pull/5144)
+- Fixed the module information button in Office365 and Github Panel tab to open the nav drawer. [#5167](https://github.com/wazuh/wazuh-kibana-app/pull/5167)
+- Fixed a UI crash due to `external_references` field could be missing in some vulnerability data [#5200](https://github.com/wazuh/wazuh-kibana-app/pull/5200)
+- Fixed TypeError in FIM Inventory using new error-handler [#5268](https://github.com/wazuh/wazuh-kibana-app/pull/5268)
+- Fixed Wazuh main menu not displayed when navigation menu is locked [#5273](https://github.com/wazuh/wazuh-kibana-app/pull/5273)
```
[the remark this answers is not in the document]
Comment (Machi3mfl) [reply CHANGELOG.md]: Solved, reverted the last rebase that add the entries in the CHANGELOG

**Review by Tostti — COMMENTED** [review]
imagen

The PR contains a huge amount of commits, is that correct? Additionaly, there are several unsigned commits

TEST:🟢 
CR: 🟢 
PR: 🟡

**Pull request closed without merging**